src/components/Header/MobileNav.tsx

```diff
@@ -50,6 +52,10 @@ export const MobileNav: FunctionComponent<MobileNavProps> = ({
                   },
                 ]}
               />
+
+              <NavLink color="blue.800" fontWeight="500" to={ROUTES.CONTRIBUTE}>
+                Contribute
+              </NavLink>
```

Comment:
The mobile nav link does not match the font color / size of the other top-level headers ("Getting Started" and "Documentation")

Reply:
<img width="322" alt="Screen Shot 2021-11-17 at 3 48 07 PM" src="https://user-images.githubusercontent.com/8749859/142300867-33447925-564d-4879-9324-34be2f7007b2.png">